internal/conf/reposource/jvm_packages.go

```diff
@@ -0,0 +1,99 @@
+package reposource
+
+import (
+	"fmt"
+	"net/url"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+
+	"github.com/sourcegraph/sourcegraph/internal/api"
+)
+
+type Module struct {
```

Comment:
This is a shared package, let's be more explicit:

```suggestion
type MavenModule struct {
```